client/post-editor/editor-revisions-list/item.jsx

```diff
@@ -5,66 +5,70 @@
 	selectRevision = () => {
-		this.props.selectRevision( this.props.revision.id );
+		this.props.selectPostRevision( this.props.revision.id );
 	};
 
 	render() {
+		const { revision, isMultiUserSite, translate } = this.props;
+		const authorName = get( revision, 'author.display_name' );
 		return (
 			<button
 				className="editor-revisions-list__button"
 				onClick={ this.selectRevision }
 				type="button"
 			>
 				<span className="editor-revisions-list__date">
-					<PostTime date={ this.props.revision.date } />
-				</span>
-				&nbsp;
-				<span className="editor-revisions-list__author">
-					{ isObject( this.props.revision.author ) &&
-						this.props.translate( 'by %(author)s', {
-							args: { author: this.props.revision.author.display_name },
-						} ) }
+					<PostTime date={ revision.date } />
 				</span>
+
+				{ authorName &&
+				isMultiUserSite && <span className="editor-revisions-list__author">{ authorName }</span> }
+
 				<div className="editor-revisions-list__changes">
-					{ this.props.revision.changes.added > 0 && (
-						<span className="editor-revisions-list__additions">
-							{ this.props.translate( '%(changes)d word added', '%(changes)d words added', {
-								args: { changes: this.props.revision.changes.added },
-								count: this.props.revision.changes.added,
+					{ revision.changes.added > 0 && (
+						<span
+							className="editor-revisions-list__additions"
+							aria-label={ translate( '%(changes)d word added', '%(changes)d words added', {
+								args: { changes: revision.changes.added },
+								count: revision.changes.added,
 							} ) }
+						>
+							<b>+</b>
+							{ revision.changes.added }
 						</span>
 					) }
 
-					{ this.props.revision.changes.added > 0 &&
-						this.props.revision.changes.removed > 0 &&
-						', ' }
-
-					{ this.props.revision.changes.removed > 0 && (
-						<span className="editor-revisions-list__deletions">
-							{ this.props.translate( '%(changes)d word removed', '%(changes)d words removed', {
-								args: { changes: this.props.revision.changes.removed },
-								count: this.props.revision.changes.removed,
+					{ revision.changes.removed > 0 && (
+						<span
+							className="editor-revisions-list__deletions"
+							aria-label={ translate( '%(changes)d word removed', '%(changes)d words removed', {
+								args: { changes: revision.changes.removed },
+								count: revision.changes.removed,
 							} ) }
+						>
+							<b>-</b>
+							{ revision.changes.removed }
 						</span>
 					) }
 
-					{ this.props.revision.changes.added === 0 &&
-						this.props.revision.changes.removed === 0 && (
-							<span className="editor-revisions-list__minor-changes">
-								{ this.props.translate( 'minor changes' ) }
-							</span>
-						) }
+					{ revision.changes.added === 0 &&
+					revision.changes.removed === 0 && (
+						<span className="editor-revisions-list__minor-changes">{ translate( 'minor' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **10** times:
`translate( 'Minor: ' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).